swagger.yaml

```diff
@@ -100,6 +80,18 @@ paths:
             text/plain:
               schema:
                 type: string
+    delete:
+      tags:
+        - auth
+      summary: Delete session
+      description: Deletes the current session
+      security:
+        - session_id: []
+      responses:
+        204:
+          description: Successfully deleted session
+        400:
+          description: Malformed request
```

Comment:
401 if no cookie in request and 422 if unrecognized RSESSID value format

Reply:
To be honest I copied it from resource management swagger (:

Reply:
400 can still occur I think if someone sends something really messed up. Like by malformed request I understand something that can barely be recognized as an HTTP request. The 422 is more for validating if the session ID length (and I think charset) is correct.

Reply:
Gotcha